dealers_choice.html

```diff
@@ -1,15 +1,104 @@
 <!DOCTYPE html>
+<html>
   <head>
     <title>Polyamory</title>
+    <style>
+      body {
+        font-family: calibri;
+        margin: 0; 
+      }
+      /* * {
+        box-sizing: border-box;
+      } */
+      body > div > div {
+        float: left;
+        padding: 12px;
+        width: 28%;
+      }
+     
+      h1 {
+        color: blue;
+        font-family: Helvetica;
+        text-align: center;
+        text-decoration: underline;
+      }
+      h2, h3 {
+        color: blueviolet;
+        text-transform: upperCase;
+      }
+      .textSections {
+        background-color: lightgray;
+        color: black;
+        font-size: 14;
+        border: 1px solid blue;
+        padding: 2rem;
+      }
+      .where {
+        font-size: 10;
+        font-style: italic;
+      }
+      /* img {
+        max-width = 20px;
+        height: auto;
+      } */
+      .nav {
+        list-style-type: none;
+        /* for site index? */
+      }
+      .links {
+        text-size: 12;
```

Comment:
i dont think `text-size` is a valid attribute- i think you're looking for `font-size` here